### PR TITLE
add test for except star

### PR DESCRIPTION
**Pull request by eikichi18**
Add test for except star

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,8 +9,10 @@
 .coverage
 .coverage.*
 .eggs/
+.idea/
 .installed.cfg
```
Comment (loechel) [inline .gitignore:13]: This file is generated, should not be touched in a pull request

File: tests/transformer/test_try.py [hunk 2]
```diff
@@ -47,6 +48,31 @@ def test_RestrictingNodeTransformer__visit_Try__2(
...
+        m('try')
+        raise ExceptionGroup("group", [IndentationError('f1'), ValueError(65)])
+    except* IndentationError:
+        m('IndentetionError')
+    except* ValueError:
+        m('ValueError')
+"""
+
+
+def test_RestrictingNodeTransformer__visit_Try__3(mocker):
```
Comment (loechel) [inline tests/transformer/test_try.py:63]: There should be a pytest marker for Python 3.11 and up

File: tests/transformer/test_try.py [hunk 3]
```diff
@@ -47,6 +48,31 @@ def test_RestrictingNodeTransformer__visit_Try__2(
...
+    """It allows try-except statements."""
+    if IS_PY311_OR_GREATER:
```
Comment (loechel) [inline tests/transformer/test_try.py:65]: That check should be replaced with the pytest marker annotation

**Review by loechel — CHANGES_REQUESTED** [review]
see specific comments for changes

File: .gitignore (hunk 1 above)
Comment (eikichi18) [reply .gitignore]: Fixed

File: tests/transformer/test_try.py (hunk 3 above)
Comment (eikichi18) [reply tests/transformer/test_try.py]: Fixed

File: tests/transformer/test_try.py (hunk 2 above)
Comment (eikichi18) [reply tests/transformer/test_try.py:68]: Fixed

File: tests/transformer/test_try.py [hunk 4]
```diff
@@ -47,6 +50,34 @@ def test_RestrictingNodeTransformer__visit_Try__2(
     ])
 
 
+TRY_EXCEPT_STAR = """
+def try_except_star(m):
+    try:
+        m('try')
+        raise ExceptionGroup("group", [IndentationError('f1'), ValueError(65)])
+    except* IndentationError:
+        m('IndentetionError')
```
Comment (perrinjerome) [inline tests/transformer/test_try.py:59]:
```suggestion
        m('IndentationError')
```
I guess this is an accidental typo (there's the same on line 76)

Comment (eikichi18) [reply tests/transformer/test_try.py:59]: ok :)

Comment (eikichi18) [reply tests/transformer/test_try.py]: Fixed

**Review by loechel — APPROVED** [review]
Now looks good for me.

@icemac please make final decission on merge

**Review by ale-rt — APPROVED** [review]
LGTM, requires PR #238 to be merged first

File: tests/transformer/test_try.py [hunk 5]
```diff
@@ -47,6 +50,39 @@ def test_RestrictingNodeTransformer__visit_Try__2(
...
+    except* RuntimeError:
+        m('RuntimeError')
+"""
+
+
+@pytest.mark.skipif(
+    not IS_PY311_OR_GREATER,
+    reason="ExceptionGroup class was added in Python 3.11.",
+)
+def test_RestrictingNodeTransformer__visit_Try__3(mocker):
```
Comment (icemac) [inline tests/transformer/test_try.py:71]:
```suggestion
def test_RestrictingNodeTransformer__visit_TryStar__1(mocker):
```

**Review by icemac — CHANGES_REQUESTED** [review]
I am not sure whether we actually need to support this new esoteric syntax: I've never had a need for it over all the years. But as it seems easy to support it, let's keep it.

The only additional change request I have is a change log entry.

**Comment by icemac** [comment]
The code looks good now. The change log entry is still requested. Maybe we have to lower the required code coverage, so the tests can pass. (Instead of waiting for #238.)

**Review by loechel — CHANGES_REQUESTED** [review]
@eikichi18 can you please also add a changelog entry in ``CHANGES.rst``,

**Comment by eikichi18** [comment]
Done

**Review by loechel — APPROVED** [review]

**Pull request merged**